pkg/mimirtool/minisdk/doc.go

```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+
+/*
+Package minisdk contains a minified version of the model found in github.com/grafana-tools/sdk.
+Most of the model parts were stripped of to reduce the unmarshaling issues caused by model evolution,
+as our objective is to just find the queries contained in the dashboards.
+
+This model could now be probably simplified, but it's left in a form compatible with the original model,
+so it would be easier to switch back if that's ever fixed.
```

Comment:
Can we include links to any particular issues or PRs that would enable us to switch back? If that's not something we'd realistically do, maybe just remove things now? It seems round-about to use a 3rd party SDK for a piece of software made by the same company.

Reply:
> Can we include links to any particular issues or PRs that would enable us to switch back?

Reverting this PR would allow us switching back, if the upstream is ever fixed.

> If that's not something we'd realistically do, maybe just remove things now? It seems round-about to use a 3rd party SDK for a piece of software made by the same company.

I agree but implementing that is out of scope of this fix IMO.